content/config/daemon/_index.md

```diff
@@ -123,8 +123,4 @@ to troubleshoot.
 Many specific configuration options are discussed throughout the Docker
 documentation. Some places to go next include:
 
-- [Automatically start containers](../containers/start-containers-automatically.md)
-- [Limit a container's resources](../containers/resource_constraints.md)
-- [Configure storage drivers](../../storage/storagedriver/select-storage-driver.md)
-- [Container security](../../engine/security/_index.md)
-- [Configure the Docker daemon to use a proxy](./systemd.md#httphttps-proxy)
+{{% sectionlinks %}}
```

Comment:
Would the section links only point to links within the same section? (Looks like some of these were in different parts of the docs, and the line above it referring to that fact)

Reply:
o, i thought it was only in the same section, I will revert this